libs/localization/src/lib/en.json

```diff
@@ -262,9 +262,14 @@
       "save": "Save"
     },
     "DropdownHeaders": {
-      "fairnessMetric": "Fairness metric",
+      "sensitiveFeature": "Sensitive feature",
```

Comment:
somehow prettier automatically sort whole file
looks like your version is kind of out-dated, do you mind merge with main branch?